src/applications/vaos/services/mocks/var/confirmed_va.json

```diff
@@ -55,89 +55,83 @@
       }
     },
     {
-      "id": "c3e3e32701223fdf389c06eae00fcb24",
+      "id": "a784f714fa1606e39dee11e3bdbb7901",
```

Comment:
I copied the staging example into our mock data to make sure it's a realistic example of these appointments